AISKU/Tests/ApplicationInsightsDeprecatedTests.ts

```diff
@@ -15,19 +19,18 @@ export class ApplicationInsightsDeprecatedTests extends TestClass {
     private clearSpy: SinonSpy;
     private _name = "ApplicationInsightsDeprecatedTests: ";
     
+    constructor() {
+        super("ApplicationInsightsDeprecatedTests");
```

Comment:
Adding a module name -- it helps when the tests are run as they will now include this prefix so when you have multiple test classes its reported correctly and not just the name of the test